### PR TITLE
Warzone 2100 cleanup

### PR DESCRIPTION
**Pull request by ROCKNROLLKID**
- Removed duplicate line.
- Question: What is the first action line suppose to be as I do not have a directory like that on my system?

### CODE REVIEW

File: cleaners/warzone2100.xml [hunk 1]
```diff
@@ -29,6 +27,5 @@
     <description>Delete the logs</description>
     <action command="delete" search="walk.all" path="~/.warzone2100*/logs/"/>
```
Comment (az0) [inline cleaners/warzone2100.xml]: In response to your question, this line is for Linux

File: cleaners/warzone2100.xml [hunk 2]
```diff
@@ -29,6 +27,5 @@
...
     <action command="delete" search="walk.all" path="$USERPROFILE\My Documents\Warzone 2100*\logs\"/>
-    <action command="delete" search="walk.all" path="$USERPROFILE\My Documents\Warzone 2100 3.1\logs\"/>
```
Comment (az0) [inline cleaners/warzone2100.xml]: This looks OK to remove, as it is redundant

File: cleaners/warzone2100.xml (hunk 1 above)
Comment (ROCKNROLLKID) [inline cleaners/warzone2100.xml]: OK. I figured something like that.

**Comment by ROCKNROLLKID** [comment]
Should be OK to merge.

**Comment by az0** [comment]
I will merge when the testing comment is preserved, please.

**Comment by ROCKNROLLKID** [comment]
Done.

**Pull request merged**